### ISSUE TITLE
Participation Event + Backoffice

### ISSUE THREAD

**Issue by pcombet-adosis** [thibaultvanc/fillingo_recettage#32]
Pour pouvoir générer le fichier Excel, je complete ma participation en tant que pcombet, participant à Naples Groupe 1. Je VALIDE ma participation, j'ai le message d'enregistrement (voir capture ci-dessous)
![image](https://user-images.githubusercontent.com/30263049/28516678-54a2d892-7062-11e7-8ec3-70f327e7cccb.png)
Je reviens en tant que gestionnaire dans le backoffice. Souscriptions : j'ai toujours 0 participant inscrit, et mon statut  est toujours "En cours"... (voir capture ci dessous)
![image](https://user-images.githubusercontent.com/30263049/28516739-8dfd16ca-7062-11e7-835a-f2325939c206.png)
REMARQUE : je me suis déconnecté, reconnecté, vidé le cache...

**Comment by thibaultvanc** [comment]
Oui, c'est une ligne du code qui était commentée. . Je l'ai remise.
